aws/connection_config.go

```diff
@@ -6,11 +6,12 @@ import (
 )
 
 type awsConfig struct {
-	Regions      []string `cty:"regions"`
-	Profile      *string  `cty:"profile"`
-	AccessKey    *string  `cty:"access_key"`
-	SecretKey    *string  `cty:"secret_key"`
-	SessionToken *string  `cty:"session_token"`
+	Regions       []string `cty:"regions"`
+	Profile       *string  `cty:"profile"`
+	AccessKey     *string  `cty:"access_key"`
+	SecretKey     *string  `cty:"secret_key"`
+	SessionToken  *string  `cty:"session_token"`
+	MaxRetryCount *string  `cty:"max_retry_count"`
```

Comment:
Various AWS SDKs use the name `MaxErrorRetry`, so I think we can use the same here, as it's also more descriptive as to what this retry count affects